specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/AnomalyDetector.json

```diff
@@ -0,0 +1,504 @@
+        "operationId": "DetectChangePoint",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Time series points and granularity is needed. Advanced model parameters can also be set in the request if needed.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/AnomalyDetectorError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Detect change point example": {
+            "$ref": "./examples/ChangePointDetect.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnomalyDetectorError": {
+      "type": "object",
+      "description": "Error information returned by the API.",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "enum": [
+            "InvalidCustomInterval",
+            "BadArgument",
+            "InvalidGranularity",
+            "InvalidPeriod",
+            "InvalidModelArgument",
+            "InvalidSeries",
+            "InvalidJsonFormat",
+            "RequiredGranularity",
+            "RequiredSeries"
+          ],
+          "x-ms-enum": {
+            "name": "AnomalyDetectorErrorCodes",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "description": "A message explaining the error reported by the service.",
+          "type": "string"
+        }
+      }
+    },
+    "TimeGranularity": {
+      "type": "string",
+      "description": "Optional argument, can be one of yearly, monthly, weekly, daily, hourly, minutely, secondly, microsecond or none. If granularity is not present, it will be none by default. If granularity is none, the timestamp property in time series point can be absent.",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "TimeGranularity",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "yearly"
+          },
+          {
+            "value": "monthly"
+          },
+          {
+            "value": "weekly"
+          },
+          {
+            "value": "daily"
+          },
+          {
+            "value": "hourly"
+          },
+          {
+            "name": "perMinute",
+            "value": "minutely"
+          },
+          {
+            "name": "perSecond",
+            "value": "secondly"
```

Comment:
>secondly [](http://example.com/codeflow?start=22&length=8)

also this does most probably not mean every second

Reply:
same as above